docs/doxygen/demos/demos_subpages.txt

```diff
@@ -33,11 +33,22 @@ Messages in this demo are sent at QoS 0, which guarantees at most one delivery a
 
 /**
 @page mqtt_demo_serializer Serializer Demo
-@brief Uses plaintext sockets to establish a TCP connection, and demonstrates QoS 0 implementation using lightweight serializer API.
+@brief Uses a POSIX socket based transport interface implementation to establish a TCP connection, and demonstrates QoS 0 implementation using only the MQTT serializer API.
 
-This demo demonstrates the subscribe-publish workflow of MQTT. After subscribing to a single topic filter, it publishes to the same topic and waits for receipt of that message to be returned from the server. It demonstrates use of the serializer API as a lightweight alternative to the standard MQTT API.
+This demo demonstrates the subscribe-publish workflow of MQTT at QoS 0 level. After subscribing to a single topic filter, it publishes to the same topic and waits for receipt of that message to be returned from the server at QoS 0 level. It demonstrates use of the serializer API as a lightweight alternative to the standard MQTT API. This cycle of publishing to the broker and receiving the same message back from the broker is repeated indefinitely.
 
 Messages in this demo are sent at QoS 0, which guarantees at most one delivery according to the MQTT spec. See the demo workflow below:
 
 @image html serializer.png width=90%
 */
+
+/**
+@page mqtt_demo_subscription_manager Susbcription Manager Demo
+@brief Uses an OpenSSL-based transport interface implementation to establish a TLS connection, and demonstrates QoS 1 implementation using a subscription manager to manage multiple subscriptions, register callbacks, and handle wildcard topics.
+
+This demo demonstrates the subscribe-publish workflow of MQTT at QoS 1 level, using a subscription manager. After subscribing to a topic filter, it publishes to the same topic and waits for receipt of that message to be returned from the server at QoS 1 level. It repeats this process for 3 different topic filters, including one wildcard filter. This cycle of publishing to the broker and receiving the same message back from the broker is repeated indefinitely.
```

Comment:
The text should also mention that the application registers different callbacks with the subscription manager for each topic filter that the application subscribes to. Also, mention that the subscription manager invokes the callback registered for topic filter that matches the topic of an incoming PUBLISH message.